### DISCUSSION TITLE
Beets should respect $XDG_MUSIC_DIR

### DISCUSSION THREAD

**Discussion by sebastianrasor** [beetbox/beets#5162] [Ideas]
By default beets is configured to use ~/Music unless otherwise specified. I think that the order of preference should be as follows:
1. directory as configured in config.yaml
2. $XDG_MUSIC_DIR if it exists
3. $HOME/Music

**Comment by Serene-Arc** [comment]
This is a good suggestion! Perhaps you could open a feature request so that we can assign work on it?

**Reply by sebastianrasor** [reply]
I went ahead and opened one here:
https://github.com/beetbox/beets/issues/5168
